test/misc.jl

```diff
@@ -6,11 +6,18 @@ using CassetteOverlay, Test
 pass = @overlaypass MiscTable
 
 # Issue #9 – Core.Compiler.return_type needs a special casing
+# TODO support invalidation mechanism for overlayed method
+# @test pass(sin, 1) do f, args...
+#     tt = Base.signature_type(f, Any[Core.Typeof(a) for a = args])
+#     return Core.Compiler.return_type(tt)
+# end == Float64
```

Comment:
I found we seem to lack the invalidation support for overlayed method definition.